tests/e2e/performance/performance_lib.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from datetime import datetime
+
+from ocs_ci.ocs.resources import pod
+
+logger = logging.getLogger(__name__)
+
+
+def write_fio_on_pod(pod_obj, file_size):
```

Comment:
@Avilir @ypersky1980  Possible to alter this function without using pod_obj ?
I meant better to change this function to start IO using just pod name, this will help us to run IO's on more pods without using obj(as you know creating more obj's we have memory issues)
@keesturam ^^

Reply:
@ramkiperiy Do you already do this as part of any of the scale test functions?

Reply:
ack

Reply:
@ramkiperiy @keesturam @Avilir  - created a new Github issue for rewrite write_fio_on_pod :+1: 
https://github.com/red-hat-storage/ocs-ci/issues/3451